src/plugins/recordTypes/intake/forms/index.js

```diff
@@ -0,0 +1,8 @@
+import defaultForm from './default';
+
+export default configContext => ({
+  default: defaultForm(configContext),
+  doorstep: {
+    disabled: true,
```

Comment:
We didn't hide the doorstep donation template, but it doesn't seem like they're even using the intake record, so it's probably fine to leave it out.